test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Ensure none of the build config inject secrets still exist in the file system
+for s in $(ls /tmp/secret? secret?); do
```

Comment:
That being said I do see how the `for` loop executing at all is counter-intuitive ... then again we're in Bash so the bar is a little lower for "sane execution"